course/pages/dagster-essentials/lesson-5/1-whats-the-definitions-object.md

```diff
@@ -12,22 +12,28 @@ Let’s use our cookie example to demonstrate. In this case, our cookie assets -
 
 ![In this example, our cookie assets, like our dry and wet ingredients, can be likened to Dagster definitions](/images/dagster-essentials/lesson-5/cookie-definitions.png)
 
-In Dagster, the `Definitions` object is located in the `definitions.py` file. If we wanted to create a `Definitions` object for the cookie definitions we have so far, it would look something like this:
+In Dagster, the `Definitions` object is located in the `definitions.py` file. If we wanted to create a `Definitions` object for the cookie definitions it would look something like this:
 
 ```python
-defs = Definitions(
-    assets=[
-        dry_ingredients,
-        wet_ingredients,
-        chocolate_chips,
-        cookie_dough,
-        chocolate_chip_cookie_dough,
-        chocolate_chip_cookies,
-    ]
-)
+import dagster_cookies.defs as defs
+
+defs = dg.components.load_defs(defs)
 ```
 
-Note that assets aren’t the only type of definition in Dagster: there are also others like resources, schedules, and sensors. We’ll go into detail on each of these in later lessons, but for now, keep this in mind.
+If we are using `dg` to layout our Dagster project, we do not need to explicitly list all the assets in our project. Dagster can automatically load most objects (we will later see there are other object types besides assets) by importing the contents of the `defs` module.
```

Comment:
Should we have an "If we are using `dg`" here, since we instruct the user to do so?